electra/binaries/unjailbreak.sh

```diff
@@ -16,11 +16,12 @@ rm /usr/lib/libsubstitute.dylib
 rm /usr/lib/libsubstrate.dylib
 rm /usr/bin/recache
 rm /usr/bin/killall
-rm /usr/share/terminfo
+rm -rf /usr/share/terminfo
 rm /usr/libexec/sftp-server
 rm -rf /System/Library/Themes
 rm -rf /Library/Frameworks/CydiaSubstrate.framework
-rm /Library/Themes
+rm -rf /Library/MobileSubstrate/DynamicLibraries
```

Comment:
this should probably be adjusted to remove the entire MobileSubstrate dir instead

Reply:
yeah, changed